public/robots.txt

```diff
@@ -1 +1,2 @@
 # See https://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
+Sitemap: https://www.hotwire.io/sitemap.xml
```

Comment:
Same here with the www, and I think this also needs to be `.xml.gz` to match the filename
```suggestion
Sitemap: https://hotwire.io/sitemap.xml.gz
```